### PR TITLE
Changes mocks and media queries

### PR DESCRIPTION
**Pull request by AlexanderBazukevich**

### CODE REVIEW

File: src/app/pages/admin-page/forms/office-form/office-form.component.scss [hunk 1]
```diff
@@ -30,12 +30,32 @@
       min-width: 0;
     }
   }
+
+  &_nowrap > .col {
+    &:last-child {
+      @media screen and (max-width: $mobile - 1) {
```
Comment (abarmina) [inline src/app/pages/admin-page/forms/office-form/office-form.component.scss]: why is -1 here?

File: src/app/shared/mocks/vendors.ts [hunk 2]
```diff
@@ -127,8 +127,8 @@ export default [
         phoneNumber: '+375 44 553-40-00',
         country: 'Беларусь',
         city: 'Минск',
-        street: 'Центральная',
-        house: '9',
+        street: 'ул. Центральная',
+        house: 'д. 9',
```
Comment (abarmina) [inline src/app/shared/mocks/vendors.ts:131]: maybe better to add "д." when joining strings? because there will always be a house number

**Review by Zairbekov — APPROVED** [review]

File: src/app/pages/admin-page/forms/office-form/office-form.component.scss (hunk 1 above)
Comment (AlexanderBazukevich) [reply src/app/pages/admin-page/forms/office-form/office-form.component.scss]: Ooops)

File: src/app/shared/mocks/vendors.ts (hunk 2 above)
Comment (AlexanderBazukevich) [reply src/app/shared/mocks/vendors.ts:131]: can be: 'ул, пер, пр-т, пр.'. About 'д.' it just for now. We can skip it even when will join: "Минск, пр. Машерова, 2, офис 4"

Comment (abarmina) [reply src/app/shared/mocks/vendors.ts:131]: ok)

**Review by abarmina — APPROVED** [review]

**Pull request merged**